### PR TITLE
Bundle dotnet-sql-cache and dotnet-user-secrets in the CLI

### PR DESCRIPTION
**Pull request by natemcmaster**
Resolves https://github.com/aspnet/DotNetTools/issues/396

Bundles additional bundled dotnet tools so users don't need to execute "dotnet install tool" to acquire aspnetcore tools.

cc @danroth27 @DamianEdwards

### CODE REVIEW

**Comment by KathleenDollard** [comment]
Is there a way for me to look at the syntax other than installing and typing help?

**Comment by natemcmaster** [comment]
> Is there a way for me to look at the syntax other than installing and typing help?

Nope. You should be able to use `dotnet install tool -g --source https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json` to get these tools from our nightly feed.

**Comment by natemcmaster** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build

**Comment by natemcmaster** [comment]
CentOS7.1 x64 Debug Build failed due to CI issues. I've reached out to netciadmin for help.

> Possible issue connecting to GitHub or cloning the repository. This could be caused by a lack of network connectivity or a lack of disk space.

File: build/BundledDotnetTools.props [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <BundledDotnetTools Include="dotnet-watch" Version="$(AspNetCoreVersion)" />
     <BundledDotnetTools Include="dotnet-dev-certs" Version="$(AspNetCoreVersion)" />
+    <BundledDotnetTools Include="dotnet-sql-cache" Version="$(AspNetCoreVersion)" />
+    <BundledDotnetTools Include="dotnet-user-secrets" Version="$(AspNetCoreVersion)" />
+    <BundledDotnetTools Include="dotnet-watch" Version="$(AspNetCoreVersion)" />
```
Comment (livarcocc) [inline build/BundledDotnetTools.props]: I think we should probably add watch to the general help menu, as it is a lot more general use than any of the other ones.

cc @KathleenDollard

Comment (livarcocc) [reply build/BundledDotnetTools.props]: By the way, we can do that in a separate PR. I am approving this one.

**Comment by natemcmaster** [comment]
CentOS failure appears to be related to [GitHub disabling old TLS crypto](https://developer.github.com/changes/2018-02-01-weak-crypto-removal-notice/)

**Comment by natemcmaster** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build

**Pull request merged**